docker-compose.yml

```diff
@@ -1,18 +1,22 @@
 web:
   build: .
+  dockerfile: Dockerfile.development
   volumes:
     - .:/champaign
```

Comment:
I’d add a `- /champaign/node_modules/` volume right here. It creates a new mount point where Docker preserves file systems isolation between the container and the host. Since it doesn’t mount anything on this volume, it keeps files from the built image.

See https://docs.docker.com/engine/reference/builder/#volume